benchmarks/graphics_pipeline/main.cpp

```diff
@@ -844,10 +882,11 @@ void ProjApp::Render()
             frame.cmd->DrawIndexed(mSkyBox.mesh->GetIndexCount());
 
             // Draw sphere instances
-            uint32_t pipeline_index = pKnobVs->GetIndex() * kAvailablePsShaders.size() + pKnobPs->GetIndex();
-            frame.cmd->BindGraphicsPipeline(mPipelines[pipeline_index]);
-            frame.cmd->BindIndexBuffer(mSphere.mesh);
-            frame.cmd->BindVertexBuffers(mSphere.mesh);
+            const size_t vbFormatIndex = pKnobVbFormat->GetIndex();
+            const size_t pipelineIndex = pKnobVs->GetIndex() * kAvailablePsShaders.size() * kAvailableVbFormats.size() + pKnobPs->GetIndex() * kAvailableVbFormats.size() + vbFormatIndex;
```

Comment:
"This is getting out of hand!" 🙃
We might want to revisit this in another PR. Keeping adding new indices and pipelines in the lest and computing the index depending on what we need is getting messy.